packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -88,12 +97,17 @@ function DefaultUI( {
 	search = true,
 	searchLabel = undefined,
 }: DefaultUIProps ) {
+	const { view } = useContext( DataViewsContext );
+	const isInfiniteScroll = view.infiniteScrollEnabled;
 	return (
 		<>
 			<HStack
 				alignment="top"
 				justify="space-between"
-				className="dataviews__view-actions"
+				className={ clsx( 'dataviews__view-actions', {
+					'dataviews__view-actions--infinite-scroll':
```

Comment:
It might be that we don't need this if we decide that the header should always be sticky even without infinite scroll being enabled (which I think would make sense)